tests/e2e/cjs/index.test.js

```diff
@@ -266,3 +266,18 @@ describe('IO', async () => {
     assert.ok(tokenCost);
   });
 });
+
+describe('ANTRegistry', async () => {
+  let registry;
+  let address = '7waR8v4STuwPnTck1zFVkQqJh5K9q9Zik4Y5-5dV7nk';
+
+  before(async () => {
+    registry = ANTRegistry.init();
+  });
+
+  it('should retrieve ids from registry', async () => {
+    const antIdsRes = await registry.accessControlList({ address });
+    const antIds = [...antIdsRes.Owned, ...antIdsRes.Controlled];
+    assert(antIds instanceof Array);
+  });
```

Comment:
sigh...think about what we're testing here. we want to validate that the response from the contract is what the SDK expects to return back to clients, so that is what we care about.


```suggestion
  const registry = ANTRegistry.init();
  const address = '7waR8v4STuwPnTck1zFVkQqJh5K9q9Zik4Y5-5dV7nk';
  it('should retrieve ids from registry', async () => {
    const affiliatedAnts = await registry.accessControlList({ address });
    assert(typeof affiliatedAnts.Owned === "array")
    assert(typeof affilatedAnts.Controller === "array")
```